### ISSUE TITLE
create-anywidget: order of "Next steps"

### ISSUE THREAD

**Issue by dvdkouril** [manzt/anywidget#475]
Hi there,
this is a small thing, and could be a matter of taste (or something else I'm not considering). Feel free to close.

With the create-anywidget script, you get the following instructions at the end:
```
Next steps:
  1: cd test
  2: git init && git add -A && git commit -m "Initial commit" (optional)
  3: npm install
  4: npm run dev
```
This means that you make the initial commit before running `npm install`. `npm install` then produces `package-lock.json`. AFAIK, it's good practice to push lockfiles to origin, so you need to make an extra second commit.

Comparing to `npm create svelte@latest`:
<img width="597" alt="Screenshot 2024-03-08 at 3 07 24 PM" src="https://github.com/manzt/anywidget/assets/2307442/ffd3e049-b5de-4420-89ad-8495f03840a2">

I can take the bullet and make this complex change, if you agree. But then again, just close if there's a reason it's the way it currently is.

**Comment by manzt** [comment]
Nope no reason! I think I actually copied from Sveltekit, so they must have changed this recently :)

**Comment by manzt** [comment]
Fixed in #476, thanks!
